MusicAppService/services/service.js

```diff
@@ -343,6 +343,70 @@ async function getSongsByPlayListId(playlistId) {
   }
 }
 
+// creates a avatar
+async function createAvatar(_id, avatarUrl) {
+  try {
+    const userData = await Users.findById({ _id });
+    const getavataruser = await Users.findByIdAndUpdate(
+      { _id: _id },
+      {
+        avatarUrl: avatarUrl
+      },
+      {
+        new: true
+      }
+    );
+    return getavataruser;
+  } catch (error) {
+    return error;
+  }
+}
+
+//get avatar
+async function getAvatar(_id) {
+  try {
+    const userData = await Users.findById({ _id });
+    return userData.avatarUrl;
+  } catch (error) {
+    return error;
+  }
+}
+
+//add comment to playlist
+async function addComment(email, comment) {
```

Comment:
controller is sending parameters _id, comment. Why service has email? From where this email will be set?
**const userComment = await UserService.addComment(
      _id,
      comment
    );**